### PR TITLE
Adds command to delete triggertemplate

### PR DESCRIPTION
**Pull request by pradeepitm12**
this adds a command to delete a given triggertemplate from a given
namespace

Signed-off-by: Pradeep Kumar <pradkuma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [x] Run the code checkers with `make check`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/cmd/triggertemplate/delete_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	tb "github.com/tektoncd/triggers/test/builder"
+
```
Comment (piyush-garg) [inline pkg/cmd/triggertemplate/delete_test.go]: remove this space

File: pkg/cmd/triggertemplate/delete.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+			s := &cli.Stream{
+				In:  cmd.InOrStdin(),
+				Out: cmd.OutOrStdout(),
+				Err: cmd.OutOrStderr(),
+			}
+
+			if err := validate.NamespaceExists(p); err != nil {
+				return err
+			}
+
```
Comment (piyush-garg) [inline pkg/cmd/triggertemplate/delete.go]: You need to add the use of [this](https://github.com/tektoncd/cli/blob/master/pkg/helper/options/delete.go#L31) here

File: third_party/VENDOR-LICENSE/github.com/hashicorp/golang-lru/go.mod [hunk 3]
```diff
@@ -1 +1,3 @@
 module github.com/hashicorp/golang-lru
+
+go 1.12
```
Comment (piyush-garg) [inline third_party/VENDOR-LICENSE/github.com/hashicorp/golang-lru/go.mod]: This is not required

File: pkg/cmd/triggertemplate/delete.go [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+func deleteTriggerTemplate(s *cli.Stream, p cli.Params, ttName string) error {
+	cs, err := p.Clients()
+	if err != nil {
+		return fmt.Errorf("failed to create tekton client")
+	}
+
+	if err := cs.Triggers.TektonV1alpha1().TriggerTemplates(p.Namespace()).Delete(ttName, &metav1.DeleteOptions{}); err != nil {
+		return fmt.Errorf("failed to delete triggertemplate %q: %s", ttName, err)
```
Comment (danielhelfand) [inline pkg/cmd/triggertemplate/delete.go]: Should this be `err.Error()` here to get the error message?

**Pull request merged**